docker/README.md

```diff
@@ -9,10 +9,52 @@ You will need to install Docker to be able to run any of the following scripts.
 Make sure you have [Docker](https://docker.com) installed and configured by
 running `docker -v`.
 
-For HPI VMs, [this
-script](https://gist.github.com/johannesvedder/0fafcbeabe8e069f96085bfedaebd9d0)
+For HPI VMs,
+[this script](https://gist.github.com/johannesvedder/0fafcbeabe8e069f96085bfedaebd9d0)
 can be used to install and setup docker.
 
+All the following commands are meant to be run from this directory, i.e. the
+`docker/` directory of the root repository, unless otherwise specified.
+
+## Quick Start with the StudyU CLI
+
+To get started quickly, run the StudyU CLI. It will interactively guide you
+through the setup process. The CLI is a wrapper around the docker-compose
+commands described in the following sections. It is located in the root
+directory of the repository and can be run with `./studyu-cli.sh`.
+
+During the configuration of the CLI, you will be asked to define the components
+you want to run. The following components are available:
+
+- `supabase-db`: A PostgreSQL database for Supabase
+- `supabase`: A self-hosted Supabase instance (needs `supabase-db`)
```

Comment:
Does it make sense to ever run these separately? If not, we could just combine them to simplify things

Reply:
Thought of this as well, but splitting it up has the advantages that Supabase can be hosted and maintained independently from the db, which brings further benefits regarding upgrading, load-balancing, etc.